src/VisualStudio/Core/Def/Implementation/Workspace/SourceGeneratedFileManager.cs

```diff
@@ -355,30 +356,29 @@ public async Task UpdateBufferContentsAsync(CancellationToken cancellationToken)
                 }
                 else
                 {
+                    var generatedDocuments = await project.GetSourceGeneratedDocumentsAsync(cancellationToken).ConfigureAwait(false);
+                    var generatedDocument = generatedDocuments.SingleOrDefault(d =>
+                                d.SourceGenerator.GetType().FullName.Equals(_generatorTypeName, StringComparison.OrdinalIgnoreCase) &&
+                                d.SourceGenerator.GetType().Assembly.GetName().Name.Equals(_generatorAssemblyName));
```

Comment:
why is fullname using OrdinalIgnoreCase, but assembly name is just using normal Equals?

Reply:
Because I wasn't paying attention. Removed the OrdinalIgnoreCase since I don't see how that was necessary in the first place.